flojoy/job_result_utils.py

```diff
@@ -17,11 +17,20 @@ def is_flow_controled(result: dict[str, Any] | DataContainer):
     return False
 
 
-def get_next_directions(result: dict[str, Any] | None) -> Optional[list[str]]:
+def get_next_directions(result: dict[str, Any] | None) -> list[str]:
     if result is None:
-        return None
-
-    return result.get(FLOJOY_INSTRUCTION.FLOW_TO_DIRECTIONS)
+        return ["default"]
+    direction = ["default"]
+    if result.get(FLOJOY_INSTRUCTION.FLOW_TO_DIRECTIONS, None) is None:
```

Comment:
Can be simplified to `if result.get(FLOJOY_INSTRUCTION.FLOW_TO_DIRECTIONS) is None:`, `.get` returns None as a fallback value by default.